### PR TITLE
dpkg 1.18.15

### PR DESCRIPTION
**Pull request by DomT4**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Closes https://github.com/Homebrew/homebrew-core/issues/7937.

### CODE REVIEW

File: Formula/dpkg.rb [hunk 1]
```diff
@@ -25,15 +25,14 @@ def install
...
-    ENV.prepend_create_path "PERL5LIB", libexec+"lib/perl5"
+    ENV.prepend_create_path "PERL5LIB", libexec/"lib/perl5"
 
     system "./configure", "--disable-dependency-tracking",
                           "--disable-silent-rules",
                           "--prefix=#{libexec}",
                           "--sysconfdir=#{etc}",
                           "--localstatedir=#{var}",
                           "--disable-dselect",
-                          "--disable-linker-optimisations",
```
Comment (DomT4) [inline Formula/dpkg.rb]: This option doesn't exist any more, to answer the question in advance 😉.

Comment (ilovezfs) [inline Formula/dpkg.rb]: Does that mean I can't ask it anyway?

**Comment by ilovezfs** [comment]
Thanks Dom

File: Formula/dpkg.rb (hunk 1 above)
Comment (ilovezfs) [inline Formula/dpkg.rb]: hehe

**Pull request closed without merging**